client/blocks/edit-gravatar/index.jsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FilePicker from 'components/file-picker';
+import FormLabel from 'components/forms/form-label';
+import {
+	getCurrentUser,
+	isCurrentUserUploadingGravatar
+} from 'state/current-user/selectors';
+import Gravatar from 'components/gravatar';
+import { isOffline } from 'state/application/selectors';
+import { localize } from 'i18n-calypso';
+import Spinner from 'components/spinner';
+import * as OAuthToken from 'lib/oauth-token';
```

Comment:
Could we avoid *? Looks like we're only using getToken anyway.

We can also give this an alias, if getToken is unclear.
```
import { getToken as getOauthToken } from 'lib/oauth-token';
```